oidc-authentication.md

```diff
@@ -41,10 +41,10 @@ extensible and modular decentralized applications based on [Linked
 Data](https://www.w3.org/DesignIssues/LinkedData) design principles.
 
 In order to allow entities to authenticate in the [Solid
-ecosystem](https://solid.github.io/specification/), this specification
-interoperably combines functionality based on widely implemented standards
-such as [OAuth 2.0](https://tools.ietf.org/html/rfc6749) and [OpenID Connect
-Core 1.0](https://openid.net/specs/openid-connect-core-1_0.html).
+ecosystem](https://solid.github.io/specification/), this specification is
+interoperably combined with [OAuth 2.0](https://tools.ietf.org/html/rfc6749)
+and [OpenID Connect Core
```

Comment:
I think original sentence made more sense. The specification itself combines existing specs rather than this specification is combined with somewhere else.

Reply:
I think I see what you mean but the original paragraph seemed a bit convoluted and touching on who should read this. That part is clarified later. Does this update address your main concern https://github.com/solid/authentication-panel/pull/71/commits/4063bd41412f330aba37106b441b23157453223b